### PR TITLE
Metrowerks overhaul

### PR DESCRIPTION
**Pull request by CelestialAmber**
-Make MSL_C match under inline deferred
-Overhaul metrowerks header files

You're very welcome

### CODE REVIEW

File: src/Dolphin/MSL_C/arith.c [hunk 1]
```diff
@@ -1,245 +1,245 @@
...
- * Address:	........
- * Size:	000198
+ * Address:	800C6150
+ * Size:	000010
  */
-void __llmod(void)
+int abs(int __x)
 {
-	// UNUSED FUNCTION
+	return (__x >> 0x1f ^ __x) - (__x >> 0x1f);
```
Comment (kiwi515) [inline src/Dolphin/MSL_C/arith.c:10]: Does `x > 0 ? x : -x` match here?

**Review by EpochFlame — CHANGES_REQUESTED** [review]
Quite a few of these files are missing the ending newlines- we should fix that before merging. Thank you for the overhaul.

**Review by EpochFlame — APPROVED** [review]

File: src/Dolphin/MSL_C/arith.c (hunk 1 above)
Comment (CelestialAmber) [reply src/Dolphin/MSL_C/arith.c:10]: It does thankfully

**Comment by CelestialAmber** [comment]
I think I'm done for now. If you really want I can get myself to do the remaining 5 files, but I kind of feel like just keeping this PR as is and doing that in another PR.

**Review by EpochFlame — APPROVED** [review]
lgtm

**Pull request merged**